geomesa-hbase/geomesa-hbase-dist/pom.xml

```diff
@@ -93,6 +93,11 @@
             <artifactId>slf4j-log4j12</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <scope>compile</scope>
```

Comment:
I think for now we can just add it and have it twice. seems that its always around. It works fine now for accumulo so I'll hold off on that.